routes/error-tracker.js

```diff
@@ -49,6 +47,39 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  const location = '([^ \\n]+):(\\d+):(\\d+)';
+  const chromeStackTraceRegex = new RegExp(`^\\s*at (.+ )?(?:${location}|\\(${location}\\))$`, 'gm');
+  const mozillaSafariStackTraceRegex = /^([^@\n]*)@(.+):(\d+):(\d+)$/gm;
+  let validException = '';
+  let match;
+  let validExceptions = [];
+  if (chromeStackTraceRegex.test(exception)) {
+    while ((match = chromeStackTraceRegex.exec(exception))) {
+      validExceptions.push(match[0]);
```

Comment:
Looks like a mistake -- this value isn't being used anywhere in this branch. Is line 69 supposed to be outside of the conditional? If so, please add a unit test for this for this case.